Source/UnitTests/Common/FloatUtilsTest.cpp

```diff
@@ -9,6 +10,8 @@
 #include "Common/BitUtils.h"
 #include "Common/FloatUtils.h"
 
+#include "../Core/PowerPC/TestValues.h"
```

Comment:
Is the `..` needed? Normally we don't use it.

Reply:
It is unfortunately necessary. The header isn't located under `Source/Core`, so we can't reference it in another way AFAIK.